README.md

```diff
@@ -51,13 +70,31 @@ We use semver on spaceduck so you can refer to a specific tag if you want instea
   <img  src="https://user-images.githubusercontent.com/32819563/111935037-3ab41d80-8a88-11eb-8797-2b6db14cbff8.png" alt="lualine replace mode screenshot">
 </center>
 
+## PACKER
+
 ```lua
 require("lualine").setup({
   options = {
     theme = "spaceduck",
   },
 })
 ```
+## LAZYVIM
+
+```lua
+return {
+    "nvim-lualine/lualine.nvim",
+    dependencies = { 'nvim-tree/nvim-web-devicons' },
+    lazy = false,
```

Comment:
On my side it works without `lazy = false` and `priority = 1000`, at least for `lualine`. If we want to keep it, shouldn't it be a priority less than `spaceduck` to make sure the theme is loaded first and then `lualine`?